internal/schemautil/service.go

```diff
@@ -590,6 +589,20 @@ func getTechnicalEmailsForTerraform(d *schema.ResourceData, field string, s *ser
 	return schema.NewSet(schema.HashResource(TechEmailsResourceSchema), techEmails)
 }
 
+func getReadReplicaIntegrationsForTerraform(integrations []service.ServiceIntegrationOut) ([]map[string]interface{}, error) {
+	var readReplicaIntegrations []map[string]interface{}
+	for _, integration := range integrations {
+		if integration.IntegrationType == "read_replica" {
+			integrationMap := map[string]interface{}{
+				"integration_type":    integration.IntegrationType,
+				"source_service_name": integration.SourceService,
+			}
+			readReplicaIntegrations = append(readReplicaIntegrations, integrationMap)
+		}
+	}
+	return readReplicaIntegrations, nil
```

Comment:
nit: This will swallow unexpected integration type. I would set enum validation for the `integration_type` field, and then the "if" can be removed.